src/dune_rules/melange/melange_rules.ml

```diff
@@ -172,6 +172,18 @@ let build_js ~loc ~dir ~pkg_name ~mode ~module_systems ~output ~obj_dir ~sctx
            ; Dep src
            ]))
 
+(** attach [deps] to the specified [alias] AND the (dune default) [all] alias.
+
+    when [alias] is not supplied, {!Melange_stanzas.Emit.implicit_alias} is
+    assumed. *)
+let add_deps_to_aliases ?(alias = Melange_stanzas.Emit.implicit_alias) ~dir deps
+    =
+  let open Memo.O in
+  let alias = Alias.make alias ~dir in
+  let dune_default_alias = Alias.all ~dir in
+  let attach alias = Rules.Produce.Alias.add_deps alias deps in
+  attach alias >>> attach dune_default_alias
```

Comment:
These could be done in parallel with `let+ … and+ …`

Reply:
Didn't realize there's a difference in parallelism. Will use `let+ ... and+ ...`

Reply:
reflected in 7c49c1d7027915cc9aff8d3a76fa58081ba84ecd